src/main/java/ua/tc/marketplace/util/mapper/CategoryMapper.java

```diff
@@ -0,0 +1,73 @@
+package ua.tc.marketplace.util.mapper;
+
+import lombok.AllArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Component;
+import org.springframework.web.server.ResponseStatusException;
+import ua.tc.marketplace.exception.model.MappingException;
+import ua.tc.marketplace.model.entity.Category;
+import ua.tc.marketplace.model.dto.CategoryDto;
+import ua.tc.marketplace.model.entity.ClassificationAttribute;
+import ua.tc.marketplace.model.dto.ClassificationAttributeDto;
+
+import java.util.stream.Collectors;
+
+@Component
+@AllArgsConstructor
+public class CategoryMapper {
```

Comment:
delete this whole calss. Learn about Mapstruct mappers and how to use them. Also see `DemoMapper` interface. Ask questions in our chat